README.md

```diff
@@ -76,7 +76,7 @@ The quick start describes how to install and use the operator `iop` CLI command.
 ### Installation
 
 ```bash
-git clone https://istio.io/operator.git
+git clone https://github.com/istio/operator.git
```

Comment:
I had to add `export GO111MODULE=on` to make `go mod` work correctly. Other than that, builds worked OOB.

Reply:
Thanks - added this to the docs.